query/stdlib/influxdata/influxdb/min_max_influxdb_test.flux

```diff
@@ -0,0 +1,23 @@
+package influxdb_test
+
+import "testing/expect"
+
+option now = () => (2030-01-01T00:00:00Z)
+
+testcase push_down_min_bare extends "flux/planner/group_min_test" {
+    expect.planner(rules: [
+        "PushDownGroupAggregateRule": 1,
+        "PushDownRangeRule": 1,
+        "PushDownFilterRule": 1,
+    ])
+    group_min_test.group_min_bare()
```

Comment:
I'm looking at the test that this test extends, and I see that it's filtering by `_value`:
https://github.com/influxdata/flux/blob/1f16d09633be04ace0b806bbaa90db59ee4ff15a/stdlib/planner/group_min_test.flux#L25

The test for `max` is similar. This doesn't seem like it would actually validate that these functions work correctly. Am I missing something?